### DISCUSSION TITLE
neues Zwischenprodukt - Brennstäbe

### DISCUSSION THREAD

**Discussion by Flagran** [Xman1109/Atomic_Overhaul#29] [Ideas]
Hey X-Man

um verschiedene Brennstoffzellen für Space und für Planeten zu erstellen kamm mir die idee Brennstäbe einzuführen. 

->

19xGrafit + 1xU235 (Pu239 oder MOX) = 4 Uranbrennstoffstäbe (Pu239 oder MOX)

4Uranbrennstoffstäbe + leere Brennstoffzelle = Uranbrennstoffzelle
1Uranberstoffstab + leere verbesserte Brennstoffzelle = SpaceUranbrennstoffzelle

das Identisch für Plutonium MOX und Thorium + vieleicht auch die aus Iridium/Holmium/Berylium

Grüße :)

**Comment by Xman1109** [comment]
Das ist eine Idee, die ich sofort umsetzen würde, wenn ich tatsächlich Grafiken machen könnte xD
Ich werde mal gucken, aber das hört sich auf jeden Fall sehr geil an.

**Reply by Flagran** [reply]
Hey Xman,
Toll das du diesee Idee magst. *G*
Grafiken kann ich leider auch nicht. :) Aber als Übergangslösung könntest du die normalen Eisenstangen in Urangrün und mit Leuchtefekt machen wenn das über die Farbeinstellungen geht. Oder die Berlystange von SE. :)
Grüße
